src/DSP/Reverter.cpp

```diff
@@ -0,0 +1,281 @@
+/*
+  ZynAddSubFX - a software synthesizer
+
+  Reverter.cpp - Reverse Delay
+  Copyright (C) 2023-2024 Michael Kirchner
+  Author: Michael Kirchner
+
+  This program is free software; you can redistribute it and/or
+  modify it under the teabs_value of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+*/
+
+#include <cassert>
+#include <cstdio>
+#include <cmath>
+#include <cstring>
+
+#include "../Misc/Allocator.h"
+#include "../Misc/Util.h"
```

Comment:
This header is not used (according to a QtCreator warning).

Reply:
and it builds fine without it. I remove it.